src/zeit/content/volume/volume.py

```diff
@@ -154,3 +155,14 @@ def retrieve_volume_using_info_from_metadata(context):
         year=context.year,
         name=str(context.volume).rjust(2, '0'))
     return zeit.cms.interfaces.ICMSContent(uniqueId, None)
+
+
+@grok.adapter(zeit.content.volume.interfaces.IVolume)
+@grok.implementer(zeit.content.cp.interfaces.ICenterPage)
+def retrieve_corresponding_centerpage(context):
+    if context.product is None or context.product.centerpage is None:
+        return None
+    uniqueId = context.product.centerpage.format(
+        year=context.year,
+        name=str(context.volume).rjust(2, '0'))
```

Comment:
Die Konfigurationsdateien (in dem Fall http://cms-backend.zeit.de:9000/cms/work/data/products.xml) sollen prinzipiell auch von "technisch versierten" nicht-Technikern bearbeitet werden, insofern versuchen wir, die möglichst "freundlich" zu halten -- die sind schon so fehleranfällig genug.